tests/integration/test_multi_objective_bayesian_optimization.py

```diff
@@ -19,6 +19,7 @@
 from trieste.acquisition import (
     BatchMonteCarloExpectedHypervolumeImprovement,
     ExpectedHypervolumeImprovement,
+    HippoPenalizationAcquisitionFunction,
```

Comment:
I would capitalize HIPPO, its an acronym

Reply:
Consider how readable are these:

```
HippoPenalizationAcquisitionFunction
HIPPOPenalizationAcquisitionFunction
```

personally I prefer the former, but maybe it's just me? Aloternatively we can just use `HIPPO` similarly to `GIBBON`

Reply:
Just HIPPO it is